packages/tools/devtools/devtools-core/api-report/devtools-core.alpha.api.md

```diff
@@ -16,6 +16,24 @@ export type ContainerKey = string;
 // @beta
 export function createDevtoolsLogger(baseLogger?: ITelemetryBaseLogger): IDevtoolsLogger;
 
+// @alpha
+export interface DataObjectProps {
+    dataObject: PureDataObject;
+    label?: string;
+}
+
+// @alpha
+export interface DecomposedContainer extends IEventProvider<IContainerEvents> {
```

Comment:
We shouldn't need to export this. This can just be an implementation detail of the library.